package.json

```diff
@@ -64,21 +64,21 @@
     "vue-loader": "15.9.8",
     "vue-template-compiler": "2.6.14",
     "webpack": "5.74.0",
-    "webpack-cli": "5.0.0",
+    "webpack-cli": "^5.0.0",
```

Comment:
Could you please remove the `^`? We manage our dependencies via dependabot so we have our dependency versions pinned to specific versions.

Reply:
Oh that was not intentional 😅